src/QuadForm/Quad/ZGenus.jl

```diff
@@ -135,20 +134,19 @@ function _p_adic_symbol(A::MatElem, p, val)
   q = p^m0
   n = nrows(A)
   A = divexact(A, q)
-  Fp = Native.GF(p)
+  Fp = GF(p)
```

Comment:
No, should be `Native.GF`

Reply:
I will revert them. What is the reason for this ?